pkg/sql/catalog/tabledesc/index.go

```diff
@@ -273,6 +273,11 @@ func (w index) ForEachColumnID(fn func(colID descpb.ColumnID) error) error {
 	return w.desc.RunOverAllColumns(fn)
 }
 
+// FirstExplicitColumnOrdinal returns the first ordinal of any explicit columns.
+func (w index) FirstExplicitColumnOrdinal() int {
+	return w.desc.FirstExplicitColumnOrdinal()
```

Comment:
This is in line with my recent changes from https://github.com/cockroachdb/cockroach/pull/58678 and the method renaming was done for the sake of naming consistency with the rest of `catalog.Index`.